tools/permission_prompt_test.py

```diff
@@ -79,10 +79,10 @@ def test_write_arg(self):
         assert stderr == b''
 
     def test_write_no(self):
-        code, stdout, stderr = self.run('needsWrite', b'N\n')
+        code, _stdout, stderr = self.run('needsWrite', b'N\n')
         assert code == 1
         # FIXME this error message should be in stderr
```

Comment:
Shouldn't this line, 103, and 122 be removed since the messages are now sent in stderr?

Reply:
Good catch. Fixed.